editor/editor.tsx

```diff
@@ -1,30 +1,31 @@
-import * as gosling from 'gosling.js';
 import React, { useRef, useState, useEffect, useCallback, useMemo } from 'react';
+import type { RouteComponentProps } from 'react-router-dom';
 import ReactMarkdown from 'react-markdown';
+import * as gosling from 'gosling.js';
 import gfm from 'remark-gfm';
 import PubSub from 'pubsub-js';
 import fetchJsonp from 'fetch-jsonp';
-import EditorPanel, { type EditorLangauge } from './editor-panel';
 import { drag as d3Drag } from 'd3-drag';
 import { event as d3Event } from 'd3-selection';
 import { select as d3Select } from 'd3-selection';
 import stringify from 'json-stringify-pretty-compact';
 import _SplitPane, { type SplitPaneProps } from 'react-split-pane';
-import ErrorBoundary from './error-boundary';
 import { debounce, isEqual } from 'lodash-es';
-import { ExampleGroups, examples } from './example';
-import { traverseTracksAndViews } from '../src/core/utils/spec-preprocess';
 import stripJsonComments from 'strip-json-comments';
 import JSONCrush from 'jsoncrush';
-import { ICONS, type ICON_INFO } from './icon';
-import { getHtmlTemplate } from './html-template';
-import { Themes } from 'gosling-theme';
-
-import type { RouteComponentProps } from 'react-router-dom';
 import type { HiGlassSpec } from '@higlass.schema';
 import type { Datum } from '@gosling.schema';
+import { Themes } from 'gosling-theme';
 
-import './editor.css';
+import { ICONS, type ICON_INFO } from './icon';
+import { getHtmlTemplate } from './html-template';
+import ErrorBoundary from './error-boundary';
+import { traverseTracksAndViews } from '../src/core/utils/spec-preprocess';
+import { examples, type Example } from './example';
+import EditorPanel, { type EditorLangauge } from './EditorPanel';
+import EditorExamples from './EditorExamples';
+
+import './Editor.css';
```

Comment:
Reorder the imports. We should consider having a eslint rule to do this automatically

Reply:
It would be good to have!